taxi/templatetags/query_transform.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def query_transform(request, **kwargs):
+    updated = request.GET.copy()
+    for k, v in kwargs.items():
```

Comment:
```suggestion
    for key, value in kwargs.items():
```

Reply:
fixed in commit '3b6145f9c0ee00c51937462d896067db1075cbe0'